src/lightning_lite/CHANGELOG.md

```diff
@@ -41,11 +41,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 -
 
 
-### Fixed
 
+## [1.8.2] - 2022-11-17
 
-- Fixed the automatic fallback from `LightningLite(strategy="ddp_spawn", ...)` to `LightningLite(strategy="ddp", ...)` when on an LSF cluster ([#15103](https://github.com/PyTorchLightning/pytorch-lightning/issues/15103))
+### Fixed
 
+- Fixed the automatic fallback from `LightningLite(strategy="ddp_spawn", ...)` to `LightningLite(strategy="ddp", ...)` when on an LSF cluster ([#15103](https://github.com/PyTorchLightning/pytorch-lightning/issues/15103))
```

Comment:
I don't think we should have a lite changelog yet

Reply:
well it is strange to release with no changes :D

Reply:
anyway, this shall be updated on release PR not here :otter: 
cc: @lantiga